Google.Cloud.Functions.Invoker/EntryPoint.cs

```diff
@@ -54,15 +50,24 @@ public static class EntryPoint
         /// </returns>
         public static async Task<int> StartAsync(Assembly functionAssembly, string[] args)
         {
+            // Clear out the ASPNETCORE_URLS environment variable in order to avoid a warning when we start the server.
+            // An alternative would be to *use* the environment variable, but as it's populated (with a non-ideal value) by
+            // default, I suspect that would be tricky.
+            Environment.SetEnvironmentVariable("ASPNETCORE_URLS", null);
+
             // TODO: Catch exceptions and return 1, or just let the exception propagate? It probably
             // doesn't matter much. Potentially catch exceptions during configuration, but let any
             // during web server execution propagate.
             var environment = FunctionEnvironment.Create(functionAssembly, args, ConfigurationVariableProvider.System);
-            var builder = Host.CreateDefaultBuilder(args)
+            IHostBuilder builder = Host.CreateDefaultBuilder(args)
                 .ConfigureWebHostDefaults(webBuilder => webBuilder
+                    .ConfigureLogging(logging =>
+                    {
+                        logging.ClearProviders(); logging.AddProvider(environment.LoggerProvider);

```

Comment:
nit: different line for adding the provider?

Reply:
Doh - absolutely. Thanks!